### PR TITLE
OADP -3768 Doc Improvement Double Stacked Admonitions ROSA AWS STS

### PR DESCRIPTION
**Pull request by CarmiWisemon**
OADP 1.3.1

OCP 4.13+

Resolves - https://issues.redhat.com/browse/OADP-3768

Deploy preview - 

https://74137--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/oadp-rosa/oadp-rosa-backing-up-applications.html#installing-oadp-rosa-sts_oadp-rosa-backing-up-applications

https://74137--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/aws-sts/oadp-aws-sts#installing-oadp-aws-sts_oadp-aws-sts-backing-up-applications

### CODE REVIEW

**Review by PrasadJoshi12 — APPROVED** [review]

File: modules/installing-oadp-aws-sts.adoc [hunk 1]
```diff
@@ -220,15 +220,15 @@ $ cat << EOF | oc create -f -
 EOF
 ----
 <1> Set this field to `false` if you do not want to use image backup.
-<2> See the first note regarding the `nodeAgent` attribute.
+<2> See the note regarding the `nodeAgent` attribute.
```
Comment (xenolinux) [inline modules/installing-oadp-aws-sts.adoc:223]: I am not sure about this but should we mention "See the important note regarding......"?

**Review by xenolinux — COMMENTED** [review]
/remove-label peer-review-needed
/remove-label peer-review-in-progress
/label peer-review-done

One question; otherwise LGTM

**Pull request merged**